specification/nhs-website-content-api.yaml

```diff
@@ -6,21 +6,20 @@ info:
   version: "Computed and injected at build time by `scripts/set_version.py`"
   description: |
     ## Overview
-    An API representation of content on the [NHS website](https://www.nhs.uk)
+    Use this API to get page content from the [NHS website](https://www.nhs.uk)
+
     You can reuse the following content from the NHS website:
-      - health conditions
-      - medicines
-      - pregnancy
-      - mental health
-      - videos
-      - common health questions
-      - information about NHS services
+      * health conditions
```

Comment:
Should 'live well' be listed in here too?

Reply:
Yes and not videos